extensions/azurecore/src/azureResource/commands.ts

```diff
@@ -46,11 +46,15 @@ export function registerAzureResourceCommands(appContext: AppContext, tree: Azur
 		const service = new AzureResourceGroupService();
 		const resourceGroups: azureResource.AzureResourceResourceGroup[] = [];
 		for (const tenant of account.properties.tenants) {
-			const tokens = await appContext.apiWrapper.getSecurityToken(account, azdata.AzureResource.ResourceManagement);
-			const token = tokens[tenant.id].token;
-			const tokenType = tokens[tenant.id].tokenType;
+			try {
```

Comment:
I know this is not the right way to do this but the problem I have now is that for one tenant, user doesn't have access to get resource groups and it makes the whole thing fail

Reply:
I'm actually going to be fixing this as part of some work I'm doing for deployments - so if you can wait for a bit this should hopefully not be a problem once that gets merged in.

Reply:
Sounds good. I can wait